README.rst

```diff
@@ -163,6 +163,51 @@ Delete
     print(response)
     # <class pybutton.Response >
 
+Accounts
+~~~~~~~~
+
+All
+'''
+
+.. code:: python
+
+    from pybutton import Client
+
+    client = Client('sk-XXX')
+
+    response = client.accounts.all()
+
+    print(response)
+    # <class pybutton.Response [2 elements]>
+
+Transactions
+''''''''''''
+
+.. code:: python
+
+    from pybutton import Client
+
+    client = Client('sk-XXX')
+
+    response = client.accounts.transactions('acc-123',
+        start='2016-07-15T00:00:00.000Z',
+        end='2016-09-30T00:00:00.000Z'
+    )
+
+    print(response)
+    # <class pybutton.Response [100 elements]>
+
+    cursor = response.next()
```

Comment:
`#next` is somewhat of a special method name in python `2.x`.  It's a part of the iteration protocol that we should try to avoid colliding with.  `#nextCursor`?